src/Modelling/Auxiliary/Output.hs

```diff
@@ -113,6 +117,17 @@ checkTaskText taskText
     usedElements = concatMap (concatMap singleton) taskText
     allElements = [minBound ..]
 
-extra :: OutputCapable m => Maybe (Map Language String) -> LangM m
-extra (Just extraMap) = paragraph $ translate $ put extraMap
-extra _ = pure ()
+data ExtraText
+  = NoExtraText
+  | Static (Map Language String)
+  | Collapsible Bool (Map Language String)
+  deriving (Data, Eq, Read, Show)
+
+extra :: OutputCapable m => ExtraText -> LangM m
+extra NoExtraText = pure ()
+extra (Static textMap) = paragraph $ translate $ put textMap
+extra (Collapsible defaultState textMap) =
+  collapsed
+    defaultState
+    (put $ M.fromList [(English, "Additional information"), (German, "Zusätzliche Informationen")])
```

Comment:
Maybe the title should also be parametrized, by adding  another argument to the `Collapsible` constructor.